control-plane/connect-inject/terminatinggatewayservice_types_test.go

```diff
@@ -0,0 +1,434 @@
+package connectinject
```

Comment:
Should this file be renamed to terminatinggatewayservice_controller_test.go ?

Reply:
Yes, done.